# fix POD errors
# most of these are misplaces/missing =over
# plus a mistyped N<..> instead of B<..>
--- a/scripts/stag-diff.pl
+++ b/scripts/stag-diff.pl
@@ -186,7 +186,9 @@ stag-diff - finds the difference between
 Compares two data trees and reports whether they match. If they do not
 match, the mismatch is reported.
 
-=over ARGUMENTS
+=head2 ARGUMENTS
+
+=over
 
 =item -help|h
 
--- a/scripts/stag-join.pl
+++ b/scripts/stag-join.pl
@@ -105,6 +105,8 @@ file
 
 =head1 ARGUMENTS
 
+=over
+
 =item -p|parser FORMAT
 
 FORMAT is one of xml, sxpr or itext, or the name of a perl module
@@ -115,6 +117,8 @@ xml assumed as default
 
 FORMAT is one of xml, sxpr or itext, or the name of a perl module
 
+=back
+
 =head1 LIMITATIONS
 
 currently not event based, so may not be memory efficicent. could be
--- a/scripts/stag-handle.pl
+++ b/scripts/stag-handle.pl
@@ -135,7 +135,9 @@ stag-handle - streams a stag file throug
 will take a Stag compatible format (xml, sxpr or itext), turn the data
 into an event stream passing it through my-handler.pl
 
-=over ARGUMENTS
+=head1 ARGUMENTS
+
+=over
 
 =item -help|h
 
--- a/Data/Stag/HashDB.pm
+++ b/Data/Stag/HashDB.pm
@@ -37,7 +37,7 @@ Used for building indexes over Stag file
 You need to provide a B<record_type> - this is the type of element
 that will be indexed
 
-You need to provide a N<unique_key> - this is a single value used to
+You need to provide a B<unique_key> - this is a single value used to
 index the B<record_type>s
 
 For example, if we have data in the stag structure below, and if ss_no
